### PR TITLE
Setup new node image with swap enabled

### PR DESCRIPTION
**Pull request by ehashman**
* Support Ubuntu and Fedora
* Tested with docker command on node

Co-Authored-By: Elana Hashman <ehashman@redhat.com>

continuation of https://github.com/kubernetes/test-infra/pull/22453 with the gubernator job fixed

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -565,3 +565,79 @@ presubmits:
...
+            - name: GOPATH
+              value: /go
+
+  - name: pull-kubernetes-node-swap-fedora
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    annotations:
+      testgrid-dashboards: sig-node-kubelet
+      testgrid-tab-name: pr-kubelet-gce-e2e-swap-fedora
+    always_run: true
```
Comment (mrunalp) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:610]: Should this be false?

Comment (ehashman) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml]: Yes, phew! fixed.

**Comment by ehashman** [comment]
/hold cancel
fixed the always_run :sweat_smile:

**Pull request merged**